index.html

```diff
@@ -8,12 +8,12 @@
     <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
-    <nav>
+    <nav  id="header">
         <p class="nav-header"> Awesome Books </p>
         <ul>
-            <li> <a href="#list" class="nav-item">List</a> </li>
-            <li> <a href="#addNew" class="nav-item">Add new</a> </li>
-            <li> <a href="#contact" class="nav-item">Contact</a> </li>
+            <li> <a id="listTab" href="#list" class="nav-item">List</a> </li>
+            <li> <a id="addNewTab" href="#addNew" class="nav-item">Add new</a> </li>
+            <li> <a id="contactTab" href="#contact" class="nav-item">Contact</a> </li>
         </ul>
```

Comment:
- [ ] Good job making the navigation bar. 👏 . However, you seem to be missing the time stamp provided on the wireframe. 
- It is meant to be consistent across the three pages. Home/List, add new and Contact. Kindly effect this change to compliment your brilliant work so far 👏 

## Your view:
![image](https://user-images.githubusercontent.com/50990925/218996046-2b425f09-5f3d-48aa-9859-a727d2cf71ca.png)

## Wireframe given
![image](https://user-images.githubusercontent.com/50990925/218996221-393c1e0f-5596-463c-a0e7-bb416af3e8d2.png)